### PR TITLE
Convert from travis to GitHub actions

### PR DESCRIPTION
**Pull request by MacFan4000**
Per https://phabricator.miraheze.org/T6457, travis-ci.org is shutting on the 31st, and travis-ci.com no longer offers a free option.

### CODE REVIEW

**Comment by MacFan4000** [comment]
A sample check run can be seen at https://github.com/MacFan4000/mw-config/runs/1532386632

**Review by RhinosF1 — APPROVED** [review]

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: .github/workflows/workflow.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php: ['7.3', '7.4', '8.0']
```
[the remark this answers is not in the document]
Comment (MacFan4000) [reply .github/workflows/workflow.yml:10]: @Universal-Omega Then why are we testing PHP 7.4 if Miraheze is running PHP 7.3?

[the remark this answers is not in the document]
Comment (Universal-Omega) [reply .github/workflows/workflow.yml:10]: Oh, good question. I do think at least PHP 8.0 should be taken out of here though.

**Review by Universal-Omega — APPROVED** [review]

**Comment by Universal-Omega** [comment]
Will revert if this does not work good. Merging to test that this will meet our needs.

**Pull request merged**